### DISCUSSION TITLE
Is there a way to uncache specific Pod ?

### DISCUSSION THREAD

**Discussion by mlch911** [swiftyfinch/Rugby#207] [Q&A] [answered]
I want to write a plugin to automaticlly uncache these changed Pods.
It's there a easy way to do so ?

**Comment by swiftyfinch** [comment]
@mlch911 Hello! What do you mean by `uncache pod`?

**Reply by mlch911** [reply]
I want to cache all remote pods + local pods by default. 
And when I change one or more local pods, I can only uncache these pods rather than rebuild all Pods.

**Reply by swiftyfinch** [reply, marked as answer]
You can cache all pods except some group of them.
And then you can work with this group in Xcode as usual without using Rugby from terminal.
